DigitalLearningSolutions.Data/Helpers/FilteringHelper.cs

```diff
@@ -122,6 +129,117 @@ FilterOptions filterOptions
                 : (items, null);
         }
 
+        public static IEnumerable<FilterOptionModel> GetPromptFilterOptions(Prompt prompt)
+        {
+            return prompt.Options.Count > 0
+                ? GetFilterOptionsForPromptWithOptions(prompt)
+                : GetFilterOptionsForPromptWithoutOptions(prompt);
+        }
+
+        public static string GetFilterValueForAdminField(
+            int promptNumber,
+            string? answer,
+            string prompt,
+            bool adminFieldHasOptions
+        )
+        {
+            var filterValueName = GetFilterValueForAdminField(promptNumber, prompt);

```

Comment:
Why is it not just `filterName`? Or `filterKey`? Normally "key" is the property you're filtering on and "value" is the value / answer you're filtering by.
Same in the rest of the new methods.